documentation/modules/ROOT/pages/configuration/topic-routing.adoc

```diff
@@ -56,79 +79,61 @@ transforms.Reroute.topic.regex=(.*)customers_shard(.*)
-----
+A Debezium change event key uses the table columns that make up the table's primary key. To route records for multiple physical tables to one topic, the event key must be unique across all of those tables. However, it is possible for each physical table to have a primary key that is unique within only that table. For example, a row in the `myserver.mydb.customers_shard1` table might have the same key value as a row in the `myserver.mydb.customers_shard2` table. 
+
+To ensure that each event key is unique across the tables whose change event records go to the same topic, the `ByLogicalTableRouter` transformation inserts a field into change event keys. By default, the name of the inserted field is `+__dbz__physicalTableIdentifier+`. The value of the inserted field is the default destination topic name.
 
-If needed, another _field name_ can be chosen by means of the `key.field.name` property
-(obviously you'll want to choose a field name that doesn't clash with existing primary key fields).
-For example the following configuration will use the name `shard_id` for the key field:
+If you want to, you can configure the `ByLogicalTableRouter` transformation to insert a different field into the key. To do this, specify the `key.field.name` option and set it to a field name that does not clash with existing primary key field names. For example: 
 
 [source]
 ----
-...
+transforms=Reroute
+transforms.Reroute.type=io.debezium.transforms.ByLogicalTableRouter
+transforms.Reroute.topic.regex=(.*)customers_shard(.*)
+transforms.Reroute.topic.replacement=$1customers_all_shards
 transforms.Reroute.key.field.name=shard_id
-...
 ----
 
-The _value_ of the field can be adjusted via the `key.field.regex` and `key.field.replacement` properties.
-The former allows you to define a regular expression that will be applied to the original topic name to capture one or more groups of characters.
-The latter lets you specify an expression that defines the value for the field in terms of those captured groups.
-For example:
+This example adds the `shard_id` field to the key structure in routed records.
+
+If you want to adjust the value of the key's new field, configure both of these options:
+
+`key.field.regex`:: Specifies a regular expression that the transformation applies to the default destination topic name to capture one or more groups of characters. 
+
+`key.field.replacement`:: Specifies a regular expression for determining the value of the inserted key field in terms of those captured groups. 
+
+For example: 
 
 [source]
 ----
-...
 transforms.Reroute.key.field.regex=(.*)customers_shard(.*)
 transforms.Reroute.key.field.replacement=$2
 ----
 
-This will apply the given regular expression to original topic names and use the second capturing group as value for the key field.
-Assuming the source topics are named `myserver.mydb.customers_shard1`, `myserver.mydb.customers_shard2` etc., the key field's values would be `1`, `2` etc.
-
-[[configuration-options]]
```

Comment:
I don't think we necessarily want to consider loosing this section because being able to quickly browse all configurable options for a given feature is useful and concise for users who may know what they're looking for.  I think its fine to reference these options in the example and various texts, but having their definition and description in a common area with other options feels better to me.  What do you think @gunnarmorling?